src/icrar/leap-accelerate/math/math_conversion.cc

```diff
@@ -95,8 +88,28 @@ namespace icrar
 
     std::vector<casacore::MVDirection> ToCasaDirectionVector(const std::vector<icrar::MVDirection>& value)
     {
-        auto res = std::vector<casacore::MVDirection>(value.size());
-        std::transform(value.cbegin(), value.cend(), res.begin(), ToCasaDirection);
-        return res;
+        return vector_map(value, ToCasaDirection);
+    }
+
+    Eigen::Vector2d to_polar(const MVDirection& xyz)
+    {
+        auto tmp = Eigen::Vector2d();
+        if (xyz(0) != 0 || xyz(1) != 0)
+        {
+            tmp(0) = std::atan2(xyz(1),xyz(0));
+        }
+        else
+        {
+            tmp(0) = 0.0;
+        }
+        tmp(1) = std::asin(xyz(2));
+        return tmp;
+    }
+
+    std::string ToMSString(std::chrono::_V2::system_clock::duration duration)
```

Comment:
If you go for a `timer` class this could be its associated `operator <<`.

Also mind that the `std::chrono::_V2` namespace is implementation-specific, you probably want to use the `std::chrono` one here.